pkgs/development/tools/continuous-integration/forgejo-actions-runner/default.nix

```diff
@@ -7,17 +7,17 @@
 
 buildGoModule rec {
   pname = "forgejo-actions-runner";
-  version = "3.0.1";
+  version = "3.1.0";
```

Comment:
Version [3.3.0](https://codeberg.org/forgejo/runner/src/branch/main/RELEASE-NOTES.md#330) is out by now, are you able to bump to that version here?